I am a genomicist interested in the evolution of genome structure and content <br/>
I primarily create tools or write repositories with lengthy explanations of pipelines and code in order that others can be inspired to use part or all of the pipeline in their own analyses <br/>

<p>
  <img alt="Linux" src="https://img.shields.io/badge/Linux-FCC624?style=flat&logo=linux&logoColor=black" />
  <img alt="bash" src="https://img.shields.io/badge/Bash-4EAA25?logo=gnubash&logoColor=fff" />
  <img alt="R" src="https://img.shields.io/badge/R-%23276DC3.svg?logo=r&logoColor=white" />
  <img alt="Markdown" src="https://img.shields.io/badge/Markdown-ffffff?style=flat&logo=markdown&logoColor=black" />
  <img alt="Stars" src="https://img.shields.io/github/stars/SAMtoBAM" />
</p>

<h3>Recent projects and pipelines</h3>
<ul>
<li><ins>Genome-graph approach to identify large transposable elements called <i>Starships</i>:</ins> <a href="https://doi.org/10.1101/2024.07.03.601904"> O'Donnell et al. 2024 <i> bioRxiv </i></a></li>
  Refer to <a href="https://github.com/SAMtoBAM/pggb_starship_pipeline"> this </a> repository for how to perform a similar genome-graph based analysis <br/>
  Refer to <a href="https://github.com/SAMtoBAM/publicgenomes-to-buscophylogeny"> this </a> repository for easily generating accurate phylogenies from public genome datasets <br/>
  Refer to <a href="https://github.com/SAMtoBAM/functional_enrichment"> this </a> repository for hypergeometric functional enrichment tests using COG annotations <br/>

<br/>
<li><ins>ScRAP dataset with phased assemblies, a population SV dataset and complex aneuploidy:</ins> <a href="https://doi.org/10.1038/s41588-023-01459-y"> O'Donnell et al. 2023 <i> Nat. Gen. </i></a></li>
  Refer to <a href="https://github.com/SAMtoBAM/aneuploidy_detection"> this </a> repository for the detection of both simple and complex aneuploidy <br/>
  Refer to <a href="https://github.com/SAMtoBAM/PhasedDiploidGenomeAssemblyPipeline"> this </a> repository for the phasing and assembling diploid heterozygotes <br/>

<br/>
<li><ins>A whole genome alignment SV calling tool:</ins> <a href="https://github.com/SAMtoBAM/MUMandCo"> MUM&Co </a> <img src="https://github.com/SAMtoBAM/MUMandCo/blob/master/logo.svg" width=15%>  <a href="https://doi.org/10.1093/bioinformatics/btaa115"> O'Donnell et al. 2020 <i> Bioinformatics </i></a></li>

<li><ins>An ensemble reference-free assembly evaluation tool:</ins>  <a href="https://github.com/SAMtoBAM/PAQman"> PAQman </a> <img src="https://github.com/SAMtoBAM/PAQman/blob/main/logo/paqman_logo_grey.png" width=13%>
  <br/>
  <br/>
<li><ins>A complete automated assembly pipeline:</ins>  <a href="https://github.com/SAMtoBAM/fusemblr"> fusemblr </a> <img src="https://github.com/SAMtoBAM/fusemblr/blob/main/logo/fusemblr.png" width=15%> </li>
<br/>
<li><ins>A free-time project to track, visualise and compare peoples visitation of</ins> <a href="https://whc.unesco.org"> UNESCO sites </a> </li>
    Refer to <a href="https://github.com/SAMtoBAM/UNESCO_data"> this </a> repository for the shiny R app and input file <br/>

</ul>
